openmetadata-ui/src/main/resources/ui/src/utils/TagsUtils.tsx

```diff
@@ -295,15 +295,18 @@ export const tagRender = (customTagProps: CustomTagProps) => {
   );
 };
 
+type ResultType = {
+  label: string;
+  value: string;
+  data: Tag;
+};
+
 export const fetchTagsElasticSearch = async (
   searchText: string,
-  page: number
+  page: number,
+  filterClassifications?: string[]
```

Comment:
What's reason we have it as string[], It will increase complexity a bit in the loop

Reply:
To filter we can have one or more than one classification so added string[].